### ISSUE TITLE
[Info] Good results on AX210 PCIe from Ziyituod

### ISSUE THREAD

**Issue by paulotex** [OpenIntelWireless/itlwm#875]
**Have You Read Our Docs**
Yes

**Are You Reporting A Bug**
No

**Environment**

- Kext Version: 2.1.0 stable and 2.2.0 alpha
- WiFi Card Model: AX210 PCIe from Ziyituod
- PCI Product ID: System Information reports:
 ```
 Card Type:	Wi-Fi  (0x8086, 0x24)
  Firmware Version:	itlwm: 2.2.0_3984afe fw: 68.01d30b0c.0
```
- macOS Version: 13.2.1

**Description**
Report on good success using this card

**Bug Report Archive**

**Kext Download Source**
Github site from you guys.

This is on a desktop hack. 
The product I have is a [AX210 PCIe from Ziyituod](https://www.amazon.com/dp/B08NJ4HTL3)

* Tested 2.1.0 itlwm with Heliport. Max speed 100MBps, on Windows it was 500MBps.
* Tested 2.2.0 alpha itlwm with Heliport. Max speed 500MBps, same as on Windows.
* Tested 2.2.0 alpha Airportitlwm. Max speed 500MBps, same as on Windows.

I am planning to stay with Airportitlwm for now, since I need it to be availeble for remote connection after booting, without any user log in. I understand that is not possible with itlwm, Heliport only loads when a user logs in? (Do correct me if I am wrong here).

I have used this for a couple of hours, so can't say much on stability yet. This machine sleeps at night and wakes every morning, is always on. In a week I can add some more info on stability.

Thanks for the great work, really happy so far...

**Comment by danhtran94** [comment]
@zxystd Rivet Killer AX1650s on Dell XPS 9300 now scannable & connected to networks from commit `6592405aeb34c2d9b29e5c5a0b56633a5d9c64e4`

**Comment by zxystd** [comment]
@danhtran94 thank you

**Comment by starlifezzz** [comment]
I second that ,thank zxystd

**Comment by khoa-nv** [comment]
Hi, thank you for your working. Is the bluetooth working?

**Comment by paulotex** [comment]
Yes, BT works well for me. I used to have a BT USB dongle and it was less reliable than this.

**Comment by khoa-nv** [comment]
> 

I'm using AX210 on xps 9570, I tried everything I can but the BT still not working. I see the System Report => Bluetooth => Discoverable is Off

**Comment by diepeterpan** [comment]
> > 
> 
> I'm using AX210 on xps 9570, I tried everything I can but the BT still not working. I see the System Report => Bluetooth => Discoverable is Off

Which SMBIOS are you using?  I have discovered some higher SMBIOS e.g. 16.x is not supported USBInjectAll.KEXT and then the Bluetooth is not working and I had to back down to 15.x

**Comment by khoa-nv** [comment]
> 

Hmm. I tried many SMBIOS from 15.x and 16.x but it doesn't working. 
Which version of MacOS and Airportitlwm.kext you're using?

I updated everything to the latest version, I also removed the BrcmBluetoothInjector.kext as the document required.

**Comment by diepeterpan** [comment]
> > 
> 
> Hmm. I tried many SMBIOS from 15.x and 16.x but it doesn't working. Which version of MacOS and Airportitlwm.kext you're using?
> 
> I updated everything to the latest version, I also removed the BrcmBluetoothInjector.kext as the document required.

On my Acer Aspire 5 8250u, I am using SMBIOS 15,4 - and had to tape the pins as indicated to get the Bluetooth to work as neither Windows, Linux nor MacOS could the Bluetooth, the Aspire BIOS were not able to switch on the Bluetooth on the card.  Also using usbinjectall.kext to ensure the Bluetooth internal USB is picked-up. Currently running Sonoma Dev Beta 5.

![93977EA1-6AD9-4789-8173-F9717136AF9A_1_105_c](https://github.com/OpenIntelWireless/itlwm/assets/15066052/d29c4766-f018-4248-86fe-b1a020b82eaf)

**Comment by khoa-nv** [comment]
> 

Thank you so much for your help. I think I need to wait some time to update the newer version.
